mux/src/termwiztermtab.rs

```diff
@@ -272,33 +275,45 @@ impl std::io::Write for TermWizTerminalRenderTty {
 }
 
 impl termwiz::render::RenderTty for TermWizTerminalRenderTty {
-    fn get_size_in_cells(&mut self) -> anyhow::Result<(usize, usize)> {
+    fn get_size_in_cells(&mut self) -> Result<(usize, usize), RenderError> {
```

Comment:
FWIW, the mux crate doesn't need to use `thiserror`; it's not a published library crate, just an application helper crate.

Reply:
Yeah, but anyhow::Error doesn't implement std::error::Error so it can't be used in this. I used thiserror just to make the inner error type, not to expose it